README.md

```diff
@@ -4,58 +4,55 @@ Phrase Count
+Fluo. Each unique document that is added causes phrase counts to be
 incremented. Unique documents have reference counts based on the number of
 locations that point to them.  When a unique document is no longer referenced
-by any location, then the phrase counts will be decremented appropriately.  
+by any location, then the phrase counts will be decremented appropriately.
 
 After phrase counts are incremented, export transactions send phrase counts to
 an Accumulo table.  The purpose of exporting data is to make it available for
 query.  Percolator is not designed to support queries, because its transactions
 are designed for throughput and not responsiveness.
 
+This example uses the Collistion Free Map and Export Queue from [Fluo
+Recipes][11].  A Collision Free Map is used to calculate phrase counts.  An
+Export Queue is used to update the external Accumulo table in a fault tolerant
+manner.
+
 Schema
 ------
 
 This example uses the following schema. 
   
-Row                   | Column                  | Value             | Purpose
-----------------------|-------------------------|-------------------|---------------------------------------------------------------------
-uri:&lt;uri&gt;       | doc:hash                | &lt;hash&gt;      | Contains the hash of the document found at the URI
-doc:&lt;hash&gt;      | doc:content             | &lt;document&gt;  | The contents of the document
-doc:&lt;hash&gt;      | doc:refCount            | &lt;int&gt;       | The number of URIs that reference this document 
-doc:&lt;hash&gt;      | index:check             | empty             | Setting this columns triggers the observer that indexes the document 
-doc:&lt;hash&gt;      | index:status            | INDEXED or empty  | Used to track the status of whether this document was indexed 
-phrase:&lt;phrase&gt; | stat:check              | empty             | Triggers observer that handles high cardinality phrases
-phrase:&lt;phrase&gt; | stat:docCount           | &lt;int&gt;       | Total number of documents the phrase occurred in
-phrase:&lt;phrase&gt; | stat:sum                | &lt;int&gt;       | Total number of times the phrase was seen in all documents
-phrase:&lt;phrase&gt; | stat:docCount&lt;int&gt;| &lt;int&gt;       | Random document count column used for high cardinality phrases
-phrase:&lt;phrase&gt; | stat:sum&lt;int&gt;     | &lt;int&gt;       | Random count column used for high cardinality phrases
-phrase:&lt;phrase&gt; | export:check            | empty             | Triggers export observer
-phrase:&lt;phrase&gt; | export:docCount         | &lt;int&gt;       | Phrase docCount queued for export
-phrase:&lt;phrase&gt; | export:seq              | &lt;int&gt;       | A sequence number used to order exports, as they may arrive out of order.
-phrase:&lt;phrase&gt; | export:sum              | &lt;int&gt;       | Phrase sum queued for export
+Row          | Column        | Value             | Purpose
```

Comment:
I think it should be noted that this is the Fluo table.  Also, showing the schema for the Accumulo table would be helpful.